package-scripts.js

```diff
@@ -3,12 +3,32 @@ const { crossEnv } = require('nps-utils')
 module.exports = {
   scripts: {
     lint: 'eslint --fix',
-    start: crossEnv('APP_DEV=true electron-forge start'),
-    package: 'electron-forge package',
-    build: 'electron-forge make',
+    start: {
+      default: crossEnv('MODE=dev electron-forge start'),
+      dev: crossEnv('MODE=dev electron-forge start'),
+      demo: crossEnv('MODE=demo electron-forge start'),
+      production: crossEnv('MODE=demo electron-forge start'),
+    },
+    package: {
+      default: crossEnv('MODE=production electron-forge package'),
+      demo: crossEnv('MODE=demo electron-forge package'),
+      production: crossEnv('MODE=production electron-forge package'),
+    },
+    build: {
+      default: crossEnv('MODE=production electron-forge make'),
+      demo: crossEnv('MODE=demo electron-forge make'),
+      production: crossEnv('MODE=production electron-forge make'),
+    },
     publish: {
-      default: 'electron-forge publish',
-      demo: 'electron-forge publish --target @electron-forge/publisher-github',
+      default: crossEnv(
+        'MODE=production electron-forge publish --target @electron-forge/publisher-github'
```

Comment:
```suggestion
        'MODE=dev electron-forge publish --target @electron-forge/publisher-github'
```
or should it be dev option instead of production?